ui/js/models/perf/alerts.js

```diff
@@ -361,5 +362,26 @@ treeherder.factory('PhAlerts', [
                     });
                 });
             },
+            findPushIdNeighbour: (dataPoint, resultSetData, direction) => {
```

Comment:
@camd How would you add this functionality, while having in mind the transition to React?

Reply:
I think this is fine for now.  As we move to ``React`` I want to convert models like this to be JavaScript ``class``es that use ``fetch``.  But unless you're up for converting this whole model to a JS class now, we can add it this way, and then convert the whole thing when we get to it.  ``findPushIdNeighbour`` *could* be a static function in a perf helper, if it is going to be used in lots of odd places.  But from the look of this, it seems appropriate here.  ``nudgeAlert`` seems correct here.  So I think your approach is fine and we'll convert this whole model later.